global/meta/models.yml

```diff
@@ -111,6 +111,13 @@ organization:
     restriction_mode: B
     default: 0
     minimum: 0
+  default_language:
+    type: string
+    read_only: true
+    description: Filled at organization initial import.
+    default: en
```

Comment:
Remove the default here, it is not used in the action anyway

Reply:
Also, we should restrict its values to the existing languages in the client, which are `en`, `de`, `it`, `es`, `ru` and `cs`

Reply:
Why should it be updated? What should happen then? I think that users may expect to update certain default stuff too. We need a concept for this.

Reply:
I removed read_only and description from the default_language field, because it should be allowed to be updated.

Reply:
Done.